### PR TITLE
Use the file picker button for selectin pre-existing audio clips

### PR DESCRIPTION
**Pull request by nirvn**
Quick UI follow up on the fresh audio of audio playback. Instead of using the gallery button to pick audio, use the file button. It's purely a visual change, but IMHO important as the audio picker isn't using the gallery.

### CODE REVIEW

File: src/qml/editorwidgets/ExternalResource.qml [hunk 1]
```diff
@@ -374,7 +374,7 @@ EditorWidgetBase {
     width: visible ? 48 : 0
     height: 48
 
-    visible: documentViewer == 0 && isEnabled
+    visible: (documentViewer == 0 || documentViewer == 3 ) && isEnabled
```
Comment (domi4484) [inline src/qml/editorwidgets/ExternalResource.qml:377]: Those values 0, 1, 2, ... for documentViewer could be declared as consts or enum for better redeability

**Review by domi4484 — APPROVED** [review]

File: src/qml/editorwidgets/ExternalResource.qml (hunk 1 above)
Comment (nirvn) [reply src/qml/editorwidgets/ExternalResource.qml:377]: @domi4484 , they are documented in a comment towards the beginning of the QML: https://github.com/opengisch/QField/blob/70a063dc9fad99043edaec18b8bd8262de4ab1db/src/qml/editorwidgets/ExternalResource.qml#L63

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 70a063dc9fad99043edaec18b8bd8262de4ab1db: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3879-217d82a3314d9f7c4ad3f4916d7e24ac6a4e7881-x86-android.apk)
</details>

**Comment by nirvn** [comment]
@domi4484 , good suggestion re constants. We can't declare that within qml itself but we can use readonly properties; done.

File: src/qml/editorwidgets/ExternalResource.qml (hunk 1 above)
Comment (suricactus) [reply src/qml/editorwidgets/ExternalResource.qml]: @nirvn ;)

**Pull request merged**